content/zh-cn/docs/reference/using-api/api-concepts.md

```diff
@@ -32,15 +32,18 @@ Kubernetes API 是通过 HTTP 提供的基于资源 (RESTful) 的编程接口。
 为了方便或者提高效率，可以以不同的表示形式接受和服务这些资源。
 
 <!--
-Kubernetes supports efficient change notifications on resources via *watches*.
+Kubernetes supports efficient change notifications on resources via
+_watches_:
+{{< glossary_definition prepend="in the Kubernetes API, watch is" term_id="watch" length="short" >}}
```

Comment:
The 'watch' term (`content/en/docs/reference/glossary/watch.md`) has not been localized, so this PR failed the build.